microsoft/testsuites/docker/dockersuite.py

```diff
@@ -261,3 +254,63 @@ def _verify_docker_engine(self, node: Node) -> None:
             "Fail to run docker run hello-world"
         ).is_equal_to(0)
         node.log.debug(f"VerifyDockerEngine: hello-world output - {result.stdout}")
+
+    @TestCaseMetadata(
+        description="""
+            Verifies a custom Docker seccomp profile is applied and enforced.
+
+            Steps:
+            1. Install Docker on the node
+            2. Create a custom seccomp profile denying chmod
+            3. Run a container with the profile; chmod should fail.
+            4. Run the same command without the profile; chmod should succeed.
```

Comment:
Please try to full fill the test case description, it helps others understand the detailed scenario.

e.g.
```
Install Docker on the target node.

Create a custom seccomp profile that explicitly denies the chmod syscall.

Start a container with this custom seccomp profile and try to run chmod; the command should fail, proving the profile is enforced.

Start another container without the custom profile and run the same chmod command; this time it should succeed, proving the restriction only comes from the profile.
```